configs/defaults/gcnv.toml

```diff
@@ -4,9 +4,14 @@ intervals_path = 'gs://cpg-common-main/references/hg38/v0/exome_calling_regions.
 exclude_intervals = ['chrM']
 # Aim for 10-50Mbp genomic coverage per shard
 interval_shards = [['chr1'], ['chr2', 'chr3'], ['chr4', 'chr5'], ['chr6', 'chr7'], ['chr8', 'chr9', 'chr10'], ['chr11', 'chr12'], ['chr13', 'chr14', 'chr15'], ['chr16', 'chr17'], ['chr18', 'chr19', 'chr20'], ['chr21', 'chr22', 'chrX', 'chrY']]
-ploidy_priors = 'gs://cpg-common-main/references/gatk-sv/hg38/v0/sv-resources/resources/v1/hg38.contig_ploidy_priors_homo_sapiens.tsv'
+#ploidy_priors = 'gs://cpg-common-main/references/gatk-sv/hg38/v0/sv-resources/resources/v1/hg38.contig_ploidy_priors_homo_sapiens.tsv'
 allosomal_contigs = ['chrX', 'chrY']
 
+# https://github.com/broadinstitute/gatk/blob/cfd4d87ec29ac45a68f13a37f30101f326546b7d/scripts/cnv_cromwell_tests/germline/cnv_germline_case_scattered_workflow.json
```

Comment:
Am I reading it correctly that this is the config used for testing? Could that be why the max events thresholds are so low (they are set for a single downsampled chrome)?

Reply:
That's what I suspect... all the files and thresholds look subsampled/super low, so I don't think they make sense to adopt directly. I'm happy to continue for now just ignoring the QC check until we can get a line on sensible exome-wide values

Reply:
I wonder if we should ping the Broad guys and ask if they have prod config we should use?